webpack.config.dev.js

```diff
@@ -22,8 +22,24 @@ module.exports = {
   module: {
     loaders: [{
       test: /\.js$/,
-      loaders: ['babel'],
-      include: path.join(__dirname, 'source')
+      include: path.join(__dirname, 'source'),
+      loader: 'babel',
+      query: {
```

Comment:
I've tried that, but seems when added the `query` data inside the `.babelrc` file, I got `react-transform` errors. I've seen some other people having this issue as well and it was fixed by adding it directly to the webpack loader query.